CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
-# CHANGELOG.md
+# Changelog
+
+All the notable changes to this project will be documented in this file.
+
+## v1.1.0 (2021-01-05)
+### Features:
+- [standardize navbar](https://github.com/Real-Dev-Squad/website-my/issues/171)
+- [add loading state for all routes](https://github.com/Real-Dev-Squad/website-my/issues/163)
+
+### Documentation:
+- [add CHANGELOG](https://github.com/Real-Dev-Squad/website-my/issues/194)
```

Comment:
Connecting issues here so that anyone who wants to know about a particular feature/fix/doc/style change, can get to know about it from the issue itself. Because one task may have 1 or more PRs, so it makes sense to link the issues.

With that being said, we'll now require issues for every feature/fix/doc/style change PR from now on. Release PRs may not have issues, we're just bumping up the version and making change to CHANGELOG

Reply:
Linking of issues makes sense 👍🏻